examples/apps/README.md

```diff
@@ -18,23 +23,24 @@ lolcat
 Ask for help for a specific app!
 
 ```
-singularity help --app fortune cowsay.img
+singularity help --app fortune cowsay.simg
 fortune is the best app
 ```
 
 Run a particular app
 
 ```
-singularity run --app fortune cowsay.img
+singularity run --app fortune cowsay.simg
+When I reflect upon the number of disagreeable people who I know who have gone
+to a better world, I am moved to lead a different life.
+		-- Mark Twain, "Pudd'nhead Wilson's Calendar"
 ```
 
 Inspect an app
 
 ```
- singularity inspect --app fortune cowsay.img 
```

Comment:
Did you mean to remove this line?  Seems like it would be good to retain the command.

Reply:
Nope that definitely looks like a mistake, since the result is still present!